packages/common/CHANGELOG.md

```diff
@@ -2,9 +2,14 @@
 
 ## Unreleased
 
+### Changes
+- Function `uleb128Decode` now correctly throws an error if the given input contains more than a single ULEB128 encoded number
```

Comment:
Isn't it more that it actually parses it correctly, as a ULEB value should be parsed?